kod/object/active/holder/nomoveon/battler/player/user/dm/admin.kod

```diff
@@ -159,6 +161,17 @@ messages:
             if Send(i,@DMCreateItemAtt,#who=self,#string=string,
                     #oWeapon=oWeapon)
             {
+               % Identify the newly-added attribute
+               if (NOT StringContain(string, admin_create_unrevealed_itematt_command))
+               {
+                  lItemAttributes = Send(oWeapon, @GetItemAttributes);
+                  lAttribute = First(lItemAttributes);
+                  if (lAttribute <> $ AND First(lAttribute) mod 2 = 0)
```

Comment:
What wizardry is this?  Definitely deserves a comment.

Reply:
LOL I guess when you put it that way, the whole item attribute compound number isn't exactly user friendly at a glance.  Wrote up what I hope explains what's going on to ultimately result in the attribute being revealed.